docs/content/preview/yugabyte-cloud/cloud-monitor/cloud-alerts.md

```diff
@@ -48,6 +48,22 @@ Open notifications are listed on the **Notifications** tab on the **Alerts** pag
 
 When the condition that caused the alert resolves, the notification dismisses automatically.
 
+## Cluster health
+
+YugabyteDB monitors the health of your clusters, based on [cluster alert](#cluster-alerts) conditions, and displays the health as either Healthy, Needs Attention, or Unhealthy based criteria as shown in the following table.
```

Comment:
Suggestion:

YugabyteDB monitors the health of your clusters based on [cluster alert](#cluster-alerts) conditions and displays the health as either Healthy, Needs Attention, or Unhealthy. The following table provides details on the criteria used to determine the cluster health.